modules/_labs/drawer/react/lib/Drawer.tsx

```diff
@@ -30,11 +30,11 @@ export interface DrawerProps extends React.HTMLAttributes<HTMLDivElement> {
   /**
    * The `aria-labelledby` of the Drawer. Set this when there is a `header` for accessibility. The `role` attribute should also be used when this attribute is present. This value should be the same as the `id` attribute of the `header` element.
    */
-  'aria-labelledby'?: string;
+  drawerArialabelledBy?: string;
```

Comment:
I'm confused by this change. Generally we avoid using the component name in its props. This was just exposing the already existing `aria-labelledby` prop extended from `React.HTMLAttributes<HTMLDivElement>` so that we could document what it's used for